app/views/products/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="d-flex justify-content-center">
+  <div class="card" style="width: 18rem;">
+    <%= image_tag "shoe.webp", class: "card-img-top" %>
+    <div class="card-body">
+      <h5 class="card-title"><%= @product.name.titleize %></h5>
+      <p class="card-text"><%= @product.description %></p>
+      <h3 class="card-text"><%= @product.price %></h3>
+      <button class="btn btn-outline-primary">Add
+        to cart
+      </button>
+    </div>
+  </div>
```

Comment:
We are repeating these lines in above file too.

Reply:
i am going to delete show, bcz we are not use show action